test/integration/targets/openssh_keypair/tasks/main.yml

```diff
@@ -67,4 +67,10 @@
     force: yes
   register: output_read_only
 
+- name: Generate privatekey7 - standard with comment
+  openssh_keypair:
+    path: '{{ output_dir }}/privatekey7'
+    comment: 'test@privatekey7'
+  register: privatekey7_result
+
```

Comment:
Maybe add another test here:
 a) copy `{{ output_dir }}/privatekey7` and its public key to `{{ output_dir }}/privatekey8`;
 b) run `openssh_keypair` again to change the comment to something else.

Then in verification, check whether the public key (and thus the private key) itself (i.e. without the comment) stayed the same, and that only the comment changed.

Reply:
ack

Reply:
I added the following test.
Create a key with comment. Change the comment of the key.
Verify that the public key has not been changed.

Currently, I'm unable to test the comment as the return value will be in the other PR.

Reply:
You could test it by looking at the output file, or by calling ssh-keygen yourself and extract the comment from its output.

Reply:
If I call ssh-keygen to show the public file, I will get just the public key without the comment.
I thought to add the test for the comment when the second PR (#59268) will get accepted.
What do you think?

Reply:
Sounds good as well.